build/pause/Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY:	build push
+
+IMAGE = pause
+TAG = 0.8.0
+
+build:	
+	./prepare.sh
```

Comment:
I think the command here needs to be updated so that it builds static with Go 1.4